packages/synapse-react-client/src/components/dataaccess/SubmissionPage/SubmissionPage.tsx

```diff
@@ -0,0 +1,542 @@
+import AccessRequirementList from '@/components/AccessRequirementList/AccessRequirementList'
+import { CancelDataAccessRequestConfirmationModal } from '@/components/dataaccess/SubmissionPage/CancelDataAccessRequestConfirmationModal'
+import { ApproveSubmissionConfirmationModal } from '@/components/dataaccess/SubmissionPage/ApproveSubmissionConfirmationModal'
+import { displayToast } from '@/components/index'
+import {
+  useGetAccessRequirementACL,
+  useGetAccessRequirements,
+  useGetAccessRequirementWikiPageKey,
+} from '@/synapse-queries/dataaccess/useAccessRequirements'
+import useGetDataAccessSubmission from '@/synapse-queries/dataaccess/useDataAccessSubmission'
+import { useGetCurrentUserProfile } from '@/synapse-queries/user'
+import { useGetUserAccessApproval } from '@/synapse-queries/dataaccess/useAccessApprovals'
+import { formatDate } from '@/utils/functions/DateFormatter'
+import { ACT_TEAM_ID } from '@/utils/SynapseConstants'
+import { Button, Skeleton, Stack, Typography } from '@mui/material'
+import Grid2 from '@mui/material/Unstable_Grid2'
+import {
+  ACCESS_TYPE,
+  FileHandleAssociateType,
+  ManagedACTAccessRequirement,
+  Submission,
+  SubmissionState,
+} from '@sage-bionetworks/synapse-types'
+import dayjs from 'dayjs'
+import duration from 'dayjs/plugin/duration'
+import { isEmpty, toLower, upperFirst } from 'lodash-es'
+import { Fragment, ReactNode, useMemo, useState } from 'react'
+import { useNavigate } from 'react-router'
+import { SynapseErrorBoundary } from '../../error/ErrorBanner'
+import MarkdownSynapse from '../../Markdown/MarkdownSynapse'
+import { UserBadge } from '../../UserCard/UserBadge'
+import UserOrTeamBadge from '../../UserOrTeamBadge/UserOrTeamBadge'
+import { FileHandleLink } from '../../widgets/FileHandleLink'
+import RejectDataAccessRequestModal from '../RejectDataAccessRequestModal'
+
+dayjs.extend(duration)
+
+export type SubmissionPageProps = {
+  /** The ID of the submission to view */
+  submissionId: string | number
+  /** If true, display reviewer controls on the page */
+  isReviewer: boolean // There is currently no API to check if a user has review permissions on a specific submission ID
```

Comment:
Added new `isReviewer` prop so we know whether we should show the reviewer UI (e.g. approve/reject buttons)